romancal/jump/jump_step.py

```diff
@@ -79,6 +85,12 @@ def process(self, input):
             max_jump_to_flag_neighbors = self.max_jump_to_flag_neighbors
             min_jump_to_flag_neighbors = self.min_jump_to_flag_neighbors
             flag_4_neighbors = self.flag_4_neighbors
+            min_sat_area = self.min_sat_area
+            min_jump_area = self.min_jump_area
+            expand_factor = self.expand_factor
+            use_ellipses = self.use_ellipses
+            sat_required_snowball = self.sat_required_snowball
+            expand_large_events = self.expand_large_events
```

Comment:
I'm not sure it's necessary to assign to separate variables here but I se it foloows what the rest of the step is doing.